docs/content/engines/overview.md

```diff
@@ -33,7 +33,7 @@ under the License.
 |                                      Flink                                      |  1.15 - 1.19  |     ✅      |      ✅      |      ✅       |  ✅(1.17+)   |        ✅        |       ✅        |        ✅        |
 |                                      Spark                                      |   3.1 - 3.5   |     ✅      |      ✅      |      ✅       |      ✅      |        ✅        |    ✅(3.3+)     |        ✅        |
 |                                      Hive                                       |   2.1 - 3.1   |     ✅      |      ✅      |      ✅       |      ❌      |        ❌        |       ❌        |        ❌        |
-|                                      Spark                                      |      2.4      |     ✅      |      ❌      |      ❌       |      ❌      |        ❌        |       ❌        |        ❌        |
```

Comment:
We can just delete this line

Reply:
done
Preview
<img width="1088" alt="image" src="https://github.com/apache/paimon/assets/29246713/e8e0dd69-dbfd-4835-a2c2-7740c5f4a578">